compel/plugins/std/log.c

```diff
@@ -13,6 +13,8 @@ struct simple_buf {
 	void (*flush)(struct simple_buf *b);
 };
 
+int print_ts_diffs = 1;
```

Comment:
Pls write a comment before this variable. It is used only in this file, so you need to mark it as static.